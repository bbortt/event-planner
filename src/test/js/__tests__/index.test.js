import React from 'react';

import { render } from '../test-utils';

import HomePage from '../../../pages';

describe('HomePage', () => {
  it('should have a loading text per default', () => {
    const tree = render(<HomePage />).container;
    expect(tree).toMatchSnapshot();
  });
});
