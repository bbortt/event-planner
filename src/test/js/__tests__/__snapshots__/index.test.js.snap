// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should have a loading text per default 1`] = `
<div>
  <div
    class="callout secondary fitContent"
  >
    <strong>
      Am Ladä..
    </strong>
    <p>
      Bitte häb hurti chli Geduld.
    </p>
  </div>
</div>
`;
